taichi/aot/module_loader.h

```diff
@@ -17,6 +18,17 @@ class RuntimeContext;
 
 namespace aot {
 
+class TI_DLL_EXPORT Field {
```

Comment:
Thanks for adding this. In the future, please think about these:

* Since Kernel already exists, the refactoring shouldn't remove it.
* Since Field doesn't exist, we don't have to add it during the refactoring.

Let's just keep this one.